extensions/reactive-messaging-http/deployment/src/test/java/io/quarkus/reactivemessaging/websocket/WsClient.java

```diff
@@ -0,0 +1,65 @@
+package io.quarkus.reactivemessaging.websocket;
+
+import java.net.URI;
+import java.util.List;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.CopyOnWriteArrayList;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.TimeoutException;
+
+import io.vertx.core.Vertx;
+import io.vertx.core.http.WebSocket;
+
+public class WsClient {
```

Comment:
I wouldn't use `Ws`. It can also mean Web Services. Same for the other classes prefixed the same way.

I know it's test classes but it's better anyway.

Reply:
I would go for WebSocketClient, it's clear and not that long.

Reply:
will do